src/github.com/getlantern/flashlight/autoupdate/autoupdate.go

```diff
@@ -48,7 +47,7 @@ func Configure(cfg *config.Config) {
 func enableAutoupdate(cfg *config.Config) {
 	var err error
 
-	httpClient, err = util.HTTPClient(cfg.CloudConfigCA, client.Addr)
+	httpClient, err = proxied.ChainedNonPersistent(cfg.CloudConfigCA)
```

Comment:
This is a good example of the syntactic change.  In keeping with @uaalto's suggestion, this API looks cleaner and also eliminates the dependency on the client package for the Addr.